movement/utils/vector.py

```diff
@@ -119,6 +119,7 @@ def cart2pol(data: xr.DataArray) -> xr.DataArray:
         data.sel(space="y"),
         data.sel(space="x"),
     )
+    phi = phi.where(rho != 0, 0)
```

Comment:
For reference, since these are floats I replaced this with an is-close approach (see e.g. [here](https://www.geeksforgeeks.org/python/comparing-floating-points-number-for-almost-equality-in-python/) for more details)